src/gtfs/repository/CIFRepository.ts

```diff
@@ -67,12 +65,9 @@ export class CIFRepository {
     `);
 
     // overlay the long and latitude values from configuration
-    return results.map(stop => {
-      const [stop_lon, stop_lat] = proj4('EPSG:27700', 'EPSG:4326', [(stop.easting - 10000) * 100, (stop.northing - 60000) * 100]);
-      stop.stop_lon = stop_lon;
-      stop.stop_lat = stop_lat;
-      delete stop.easting;
-      delete stop.northing;
+    return results.map(row => {
+      const [stop_lon, stop_lat] = proj4('EPSG:27700', 'EPSG:4326', [(row.easting - 10000) * 100, (row.northing - 60000) * 100]);
+      const {easting, northing, ...stop} = {...row, stop_lon, stop_lat};
```

Comment:
we could combine this with the line below I think. The Object.assign stuff was done before the rest/spread things were added.

Reply:
I have pushed a new commit for that; however, a warning suppression is required as we are deliberately overwriting properties.

Reply:
leave it as two

Reply:
> leave it as two

OK I have reverted the line